src/content/app/genome-browser/components/browser-location-indicator/BrowserLocationIndicator.tsx

```diff
@@ -51,7 +54,11 @@ export const BrowserLocationIndicator = (props: Props) => {
     <div className={className}>
       <div className={styles.chrLabel}>Chromosome</div>
       <div className={styles.chrLocationView} {...onClickProps}>
-        <div className={styles.chrCode}>{chrCode}</div>
+        {genomeKaryotype && genomeKaryotype[0].is_circular ? (
```

Comment:
We can do better than this. Rather than inspecting the first chromosome from the karyotype, we should look at the one that is actually shown in the genome browser. You can find it in the redux store at browser → browserGeneral → actualChromosomeLocations:

![image](https://user-images.githubusercontent.com/6834224/149544032-9b0e1892-d343-4b85-af09-29f40f173bd7.png)

One problem though: e. coli is currently broken in genome browser; so the genome browser doesn't show anything in the canvas and can't report anything about its location in the E.coli genome. See https://github.com/Ensembl/ensembl-dauphin-style-compiler/issues/23 I suspect it may block the work on this PR.

Reply:
e coli should only be broken if there's a focus object as I understand it. This shouldn't block testing the indicator, I don't think.

Reply:
Thanks Dan and Andrey. I have updated the code.